Changelog.rst

```diff
@@ -138,6 +140,9 @@ pupmod-simp-aide
 ^^^^^^^^^^^^^^^^
 
 * Added /etc/logrotate.simp.d to default rules.
+* Ensure that the ``package`` install comes before dependent ``exec``
+  statements.
+* Allow the ``cron`` command to be customized.
```

Comment:
The update to allow cron to be customized hasn't completely been done (SIMP-5085).  So, this should be more explicit

Reply:
NB this is OK.  I thought this was about pupmod-simp-logrotate, not pupmod-simp-aide